cluster/gce/util.sh

```diff
@@ -371,6 +371,23 @@ function detect-master() {
   echo "Using master: $KUBE_MASTER (external IP: $KUBE_MASTER_IP)" >&2
 }
 
+# Detect a legacy ABAC policy file at /etc/srv/kubernetes/abac-authz-policy.jsonl
+# and set ABAC_AUTHZ_FILE if it exists
+#
+# Vars set:
+#   ABAC_AUTHZ_FILE
+function detect-legacy-abac-file() {
+  # This is the location GCE scripts wrote ABAC policy files to prior to 1.6
+  local -r legacy_abac_path="/etc/srv/kubernetes/abac-authz-policy.jsonl"
```

Comment:
really? both places I saw it referenced by the apiserver it was at this path

https://github.com/kubernetes/kubernetes/commit/c1afc4a3d842a7f3798d98cf3466aa1310e2ee7d#diff-82d9e3b777d0e39e2fea4985a20f307eL715

https://github.com/kubernetes/kubernetes/commit/ecd23a02176c5f1aa5416316f69c5601333ea421#diff-6e3b476b9225d1213dc6ad13e453fc16L789